noxfile.py

```diff
@@ -1,75 +1,21 @@
 import nox
-from pathlib import Path
-from yaml import safe_load
 
 nox.options.reuse_existing_virtualenvs = True
 
-# Parse the environment files we'll need later
-environment = safe_load(Path("environment.yml").read_text())
-conda = environment.get("dependencies")
-requirements = conda.pop(-1).get("pip")
-build_command = ["-b", "html", "docs", "docs/_build/html"]
 
-
-@nox.session(venv_backend="conda")
-def build(session):
-    _install_environment(session)
-    session.run("yarn", "build")
-
-
-@nox.session(venv_backend="conda")
+@nox.session
 def docs(session):
-    _install_environment(session)
-    session.cd("docs")
-    session.run("make", "html")
+    session.install(".[doc]")
+    session.run("sphinx-build", "-b=html", "docs/", "docs/_build/html")
 
 
-@nox.session(name="docs-live", venv_backend="conda")
+@nox.session(name="docs-live")
```

Comment:
How the "conda" removal here is impacting the docs-live?

Reply:
I guess I need to understand a little bit more about sphinx-theme-builder...

Reply:
We were using conda because we needed to get `nodejs` in our build environment, and `conda` was the easiest way. However, the `sphinx theme builder` comes with its own nodejs install scripts, so we can just use straight up `pip`

Reply:
Given the ecosystem we are in and I am pretty sure there are a lot of conda (and pydata_theme) users that would find it "awkward"(maybe that is not the proper word...) not being able to manage the nodejs we want to use here. This is also connected with some previous questions from @jorisvandenbossche in another comment.
I would like to hear the story for those conda users in this new builder context... and try/experiment with it 😉 .

Reply:
Just to be explicit: I am such an all-conda person :) (and I would still use a conda env for developing this theme, even if I don't have to install node manually anymore), and I tested this PR locally with a conda environment. 

It was first not fully clear to me how sphinx-theme-builder manages the environment (because with the addition of nox I had to think this through how to combine it with an existing conda env), but it's actually quite simple: sphinx-theme-builder doesn't do that at all, it just uses your existing environment in which it is installed when building / serving the docs. At least for the python dependencies, it's only for compiling the css/js assets that it uses the python `nodeenv` package to create a separate node environment. 

Since this is automatically handled by sphinx-theme-builder, I personally don't really care that it is a separate environment, instead of the calling conda environment (it's still using an environment, so not "polluting" the system). I _think_ that will be the case for most of the users of this package that are using conda. 
But so even if you want to ensure that sphinx-theme-builder uses the node that you installed in the conda env, you can actually do this by installing the exact version that is needed (which is specified in `pyproject.toml`: `node-version = "14.18.1"`). sphinx-theme-builder will automatically check if the version matches, I think (this can use some better documentation).

Reply:
I opened https://github.com/pradyunsg/sphinx-theme-builder/issues/6 on the sphinx-theme-builder side to better document this.

I also tried to manage nodejs with a conda env, and ran into 2 problems:

- The node version as it is pinned right now (latest 14.* version, i.e. 14.18.1) is not actually available on conda-forge
- After changing the version we need in pyproject.toml to match what I could install with conda-forge (14.17.4), I still ran into the issue that nodeenv detects my actual system-wide node installation instead of the conda-activated one. For this I opened https://github.com/pradyunsg/sphinx-theme-builder/issues/7

Reply:
Yea, I need to do documentation and tests for the tool still. I've basically been working toward "make it work well", and once that's done, I'll connect the dots of how polished the implementation is.

I've been meaning to come to doing it, now that I know this works for Furo and this theme. My free time has been scarce lately, so... it's stuck. 😅